Remove 'Create Game' and include as part of round. (#3)
ShawnSidoruk/slate: source/index.html.md

-X POST \
-d '{
  "external_round_id": "partner-round-id-1",

>   "external_game_id": "ePz8u6hbncrNzFRCqLbX8e"

Reviewer: ```suggestion
  "external_game_id": "ePz8u6hbncrNzFRCqLbX8e",
```

  "external_game_label": "Starburst"
}' https://ns-api-staging.betable.com/lfs/v100\
/users/QEaakM0qumTJd5tj/rounds
```
`POST /users/{user_id}/rounds`

This method creates a round. A round is a container used for grouping together wagers and payouts that are associated to a user. Any combination of wagers and payouts can be contained within a round, as long as the round has not been closed.

A round can consist of zero or more Wagers, and zero or more Payouts. The choice of how many Wagers and Payouts to include in a round is up to the integration partner.

<aside>

> Wagers and Payouts will be presented back to the player using the <code>external_game_label</code> provided by the Round.

Reviewer: ```suggestion
Wagers and Payouts will be presented back to the player using the <code>external_game_label</code> provided for the Round.
```

-d '{
  "external_round_id": "partner-round-id-1",
  "game_id": "ePz8u6hbncrNzFRCqLbX8e"
  "game_label": "Starburst"
}' https://ns-api-staging.betable.com/lfs/v100\
/users/QEaakM0qumTJd5tj/rounds
```
`POST /users/{user_id}/rounds`

This method creates a round. A round is a container used for grouping together wagers and payouts that are associated to a user. Any combination of wagers and payouts can be contained within a round, as long as the round has not been closed.

> A round can consist of zero or more Wagers, and zero or more Payouts. The choice of how many Wagers and Payouts to include in a round is up to the integration provider.

Reviewer: ```suggestion
A round can consist of zero or more Wagers, and zero or more Payouts. The choice of how many Wagers and Payouts to include in a round is up to the integration partner.
```

<aside>

> Wagers and Payouts will be presented back to the player using the <code>game_label</code> provided by the Round.

Reviewer: ```suggestion
Wagers and Payouts will be presented back to the player using the <code>game_label</code> provided for the Round.
```

| Parameter | Type | Example | Description |
| ---| ---| ---| --- |
| amount | Money | 100.00 | The user's balance to a maximum of 2 decimal places. |
| currency | string | USD | Currency.  When relevant this will reference ISO-4217 3-letter code. |

<aside class="comment">

> Currency code <code>XXX</code> will be used to represent free money funds.

Reviewer 2: ```suggestion
Currency code <code>XXX</code> will be used to represent simulated currencies.
```

ficticious?

</aside>

<aside class="comment">

> Currencies are associated to accounts during creation of the integration.

Reviewer: "creation of the integration" doesn't make sense to me and might need rewording.

Author: ```suggestion
Possible currencies returned are setup within the Betable API during the initial phases of partner integration
```
